release/train_tests/benchmark/config.py

```diff
@@ -13,19 +13,20 @@ class DataloaderType(enum.Enum):
 class DataLoaderConfig(BaseModel):
     train_batch_size: int = 32
     validation_batch_size: int = 256
-    prefetch_batches: int = 1
 
 
 class RayDataConfig(DataLoaderConfig):
     # NOTE: Optional[int] doesn't play well with argparse.
     local_buffer_shuffle_size: int = -1
+    ray_data_prefetch_batches: int = 4
```

Comment:
default to 1?

Reply:
nit

```suggestion
    prefetch_batches: int = 4
```

Reply:
how do I override this parameter from CLI?  is it `--ray-data-prefetch-batches=1`? 
added the prefix to make CLI args more clear

Reply:
oh right. this works then